int32 id
semap_msgs/Point3DModel model
---

